src/App.js

```diff
@@ -16,24 +16,26 @@ class App extends Component {
 			currentUser: '',
 			id: '',
 			foundMovie: '',
+			ratings: []
 			//maybe consider Nan lets do research
 		};
 	}
 
 	logInUser = user => {
-		console.log(window.location);
 		this.setState({ currentUser: user.user.name, id: user.user.id });
+		this.getUserRatings();
 		// this.setState(user.user) potential refactor later
 	};
 
 	logOutUser = () => {
 		this.setState({ currentUser: '', id: '' });
+		window.location.pathname = '/'
 	};
 
 	findMovieShowInfo = () => {
-		let movieID = window.location.pathname.split('/')
+		let movieID = parseInt(window.location.pathname.split('/')[2]);
```

Comment:
Think about how you can use render props (like the `match` object) to utilize this function in a dynamic route!

Reply:
I'll look into this!